config/releases.exs

```diff
@@ -216,4 +224,13 @@ config :service_orchestrate, Orchestrate.Application,
   ]
 
 # SERVICE ACQUIRE
-config :service_acquire, Acquire.Db.Presto, presto: Keyword.put(presto_db, :user, "acquire")
+config :service_acquire, AcquireWeb.Endpoint,
+  http: [:inet6, port: String.to_integer(System.get_env("PORT") || "4000")],
+  url: [host: "localhost"],
```

Comment:
Same `System.get_env` callout here as far as the default for the hostname?

Reply:
This is just used when generating URLs from the backend. Looks like I should remove the config entirely to remove potential confusion.

Reply:
👍